docs/reference/expectations/result_format.md

```diff
@@ -298,71 +222,78 @@ The summary `result` includes:
         "partial_unexpected_list" : [A list of up to 20 values that violate the expectation] (also in `BASIC`)
         'missing_count': The number of missing values in the column
         'missing_percent': The total percent of missing values in the column
-        'partial_unexpected_counts': [{A list of objects with value and counts, showing the number of times each of the unexpected values occurs}]
-        'partial_unexpected_index_list': [A list of up to 20 of the indices of the unexpected values in the column]
+        'partial_unexpected_counts': [{A list of objects with value and counts, showing the number of times each of the unexpected values occurs}
+        'partial_unexpected_index_list': [A list of up to 20 of the indices of the unexpected values in the column, as defined by the primary key column]
     }
 }
 ```
 
 For example:
+```python name="result_format_summary_example_set"
+```
 
+Will return the following output:
 ```python
+>>> print(validation_result.success)
+False
+
+>>> print(validation_result.result)
 {
-    'success': False,
-    'result': {
-        'element_count': 36,
-        'unexpected_count': 6,
-        'unexpected_percent': 0.16666666666666666,
-        'unexpected_percent_nonmissing': 0.16666666666666666,
-        'missing_count': 0,
-        'missing_percent': 0.0,
-        'partial_unexpected_counts': [{'value': 'A', 'count': 1}, {'value': 'E', 'count': 5}],
-        'partial_unexpected_index_list': [0, 10, 11, 12, 13, 14],
-        'partial_unexpected_list': ['A', 'E', 'E', 'E', 'E', 'E']
-    }
+    "element_count": 15,
```

Comment:
Should this example (and others for validation_result.success and .result) come from snippets too? That way they are under test and then change automatically if the test script changes? Maybe an assert statement in the script below the script tag if we don't want to show the assert in the docs? E.g. 
```
<snippet name="some name">
expected_validation_result = {
   ... (fill this in)
}
</snippet>
assert validation_result.result == expected_validation_result
```

Reply:
![Screen Shot 2023-01-09 at 11 18 52 PM](https://user-images.githubusercontent.com/25670697/211486492-c1985bf3-410d-4e9e-9a6a-789c08c5a9bd.png)

I had a stab at adjusting the output, and there is potentially a small inconsistency (in the formatting) that this introduces. It's pretty minor, so let me know if which direction you would like to go.  I'm happy either way.

Reply:
How do we do this in other docs? I don't recall the repl style `>>>` in other docs. Maybe we can just show the assert statement? like:
```
assert validation_result.success
assert validation_result.result == {"observed_value": 3.6666666666666665}
```
I see we do that in a few places e.g. https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/advanced/how_to_configure_a_dataconnector_for_splitting_and_sampling_tables_in_sql/#splitting-and-sampling

Reply:
Great idea. I think this was the result of tension between preserving the existing docs as much as possible (which used the `>>>`) and making them accurate. I like the idea of displaying the `assert`. Update coming up ;)

Reply:
![Screen Shot 2023-01-10 at 12 24 15 PM](https://user-images.githubusercontent.com/25670697/211654493-062c3219-a75e-4bfa-b29e-fc579b3f08d3.png)
Boom